python/podio_class_generator.py

```diff
@@ -359,29 +375,35 @@ def _preprocess_datatype(self, name, definition):
     data = deepcopy(definition)
     data['class'] = DataType(name)
     data['includes_data'] = self._get_member_includes(definition["Members"])
+    data['includes_data_jl'] = self._get_member_includes(definition["Members"], julia=True)
```

Comment:
Is this necessary? In the templates I don't see that we use `includes_data_jl`

Reply:
I see they are used in the templates now. Is there a reason why they have to go to a different list than the usual `includes_jl`?

Reply:
I think they can go in the same list. I will change it in the next commit.